src/server/handle-preview.js

```diff
@@ -0,0 +1,27 @@
+import fetch from 'isomorphic-fetch'
+import winston from 'winston'
+import { errorObject } from '../utilities/logger'
+
+export default ({ server, config }) => {
+
+  server.route({
+    method: 'GET',
+    path: '/api/preview/v1/{query*}',
+    handler: (req, reply) => {
+
+      const base = `${config.siteUrl}/wp-json/revision/v1`
+      const path = `${req.params.query}${req.url.search}`
+      const remote = `${base}/${path}`
+
+      winston.log('debug', `Server loading API response over HTTP for ${remote}`)
```

Comment:
Let's handle these logs in a separate ticket, we need to create some conventions